scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -151,6 +151,51 @@ namespace <%= namespace %> {
             } catch(NoQueryException exception) {}
         }
 
+        private void SetShippingAddress(MailingAddressInput mailingAddressInput, Action<List<string>, string> callback) {
```

Comment:
Is the plan that the native module will send a JSON object via a message that can be used to create a `MailingAddressInput`?